### ISSUE TITLE
Write the report

### ISSUE THREAD

**Issue by Lizzy-Pan** [Hilary777/2020-GPW-L5K1-D#9]

**Comment by Hilary777** [comment]
Divide the work according to the exam requirements
1：introduction /Conclusion/agile development
2:introduction web and function
3:database
4: moral issue

**Comment by beckyzhang123** [comment]
Group members complete their own parts according to the division of labor.

**Comment by beckyzhang123** [comment]
Is the report completed by the group members together and then submitted? Is there a requirement for the file name? @jslemmer

**Comment by jslemmer** [comment]
This is a group report. Only 1 student of your group has to submit all the files.
No need for each student to submit the same thing separately.

As far as file requirement, please use your group name as the name of your document.
Example: 2020-GPW-L5K1-D Report.docx

**Comment by Lizzy-Pan** [comment]
The team report has been basically completed.

**Comment by Hilary777** [comment]
![QQ图片20200524192722](https://user-images.githubusercontent.com/45256981/82752995-bf94c300-9df4-11ea-8502-280b4f90d345.png)
The picture is about GPW assessment semester1. I want to know something it as this picture for this semester . But I can not find GPW assessment semester 2 on student website. Can you help me? @jslemmer

**Comment by jslemmer** [comment]
The assessment file is just at the bottom of the GPW overview

![image](https://user-images.githubusercontent.com/16065891/82769189-0a025800-9e66-11ea-8248-e9170600651c.png)

**Comment by Hilary777** [comment]
Thank you for your reply. I know this. But I mean that we finally need to upload a zip to the student website? And the zip include report, database, code? @jslemmer

**Comment by jslemmer** [comment]
Yes, you will also need to upload a .zip file with everything on the student website. You should be able to see the deadline and the place to upload the files on the student website now.
